PowerShell Scanners/LLMNR Enabled/LLMNR Query.xml

```diff
@@ -0,0 +1,46 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<AdminArsenal.Export Code="PDQInventory" Name="PDQ Inventory" Version="19.1.96.0" MinimumVersion="19.0">
+  <ScanProfile>
+    <Collections type="list">
+      <ScanProfileCollection>
+        <CollectionId value="2135" />
+        <ScanProfileId value="null" />
+      </ScanProfileCollection>
+    </Collections>
+    <Scanners type="list">
+      <Scanner>
+        <ModifiedDate>2020-12-23T16:47:01.0000000-05:00</ModifiedDate>
+        <Name>multicast-query</Name>
+        <UID>3cf796f8c56a445d86f8f0447675dbc2</UID>
+        <Script>
+$Multicast = Get-ItemProperty -Path 'HKLM:SOFTWARE\Policies\Microsoft\Windows NT\DNSClient\' -Name 'EnableMulticast'
+
+
+if( $Multicast.EnableMultiCast -eq 0){
+  $Enabled = $false
+} else{
+    $Enabled = $true
+}
+
+[PSCustomObject]@{
+    Host = $env:COMPUTERNAME
+    Enabled  = $Enabled
+    PSPath = $Multicast.PSPath
+} </Script>
```

Comment:
```suggestion
        <Script>C:\PowerShell-Scanners\PowerShell Scanners\LLMNR Enabled\LLMNR Enabled.ps1</Script>
```

This should reference the script associated with the scanner and not use inline code.
In addition the script file should use the same name as the directory that holds it, you'll have to rename `multicast-query.ps1` to `LLMNR Enabled.ps1` and once that and this are done we can merge this.